packages/compass-components/src/typings.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@leafygreen-ui/form-footer';
```

Comment:
this packages does not have type definition and in order to avoid [this pipeline error](https://github.com/mongodb-js/compass/runs/4702604527?check_suite_focus=true#step:7:314), we need it.

Reply:
I would suggest that we ask about these things in #leafygreen-ui channel to clarify why this is happening and to make them aware, this does look like an issue that we should ideally not need to work around on our side

Reply:
Reported. Jira ticket: [PD-1717](https://jira.mongodb.org/browse/PD-1717)

Reply:
Is it cool if we leave a comment with that ticket linked so we know if we know if we can remove it later on when someone sees it and it's fixed?